### ISSUE TITLE
[Feature Request] new color scheme based on similarity (a la Uniprot)

### ISSUE THREAD

**Issue by camilogarciabotero** [moshi4/pyMSAviz#2]
Hi,

This tool is super! I was wondering if the tool can also have a color scheme based on the similarity of the characters, like the example below:

![image](https://user-images.githubusercontent.com/25586362/202766555-0496dfd0-3d9b-4630-ab42-b0f190df6c95.png)

**Comment by moshi4** [comment]
Hi, @camilogarciabotero 

Thank you for your valuable feature request.

I have implemented the requested feature (`Identity` Color Scheme) on a trial basis and would like you to check it out a bit.

**Example 1.**

```python
from pymsaviz import MsaViz

msa_file = "./msa_ident_color.fa"
mv = MsaViz(msa_file, color_scheme="Identity", show_label=False)
fig = mv.plotfig()
```
![msa_ident_color_example1](https://user-images.githubusercontent.com/41852815/202833913-43dda6c4-e3d4-4147-9b31-969ff0e54261.png)

**Example 2.**

```python
from pymsaviz import MsaViz

msa_file = "./msa_ident_color.fa"
mv = MsaViz(msa_file, color_scheme="Identity", show_label=False)
mv.set_plot_params(identity_color="tomato", identity_color_min_thr=40)
fig = mv.plotfig()
```
![msa_ident_color_example2](https://user-images.githubusercontent.com/41852815/202833917-fa60a488-9cdc-468f-9ae3-96c610618f0a.png)

The coloring definition may be a little different than the example you provided, but I think it looks roughly the same.

**Comment by moshi4** [comment]
I have newly released v0.2.1 with additional functionality for the `Identity` Color Scheme. Please give it a try.

Thanks for your help in improving the functionality :smiley:

**Comment by camilogarciabotero** [comment]
Thank you! It looks awesome. I'll check it out.
